recipe/patches/0035-pypy-distutils-scheme.patch

```diff
@@ -1,12 +1,7 @@
-# Branch conda
-# Node ID 85c317f36cd8fc1962c1776a6cfb1df3572c518c
-# Parent  e9052b361691a2de3cfb86712ecd4609ac3aff12
-conda schema fixes for distutils
-
-diff -r e9052b361691 -r 85c317f36cd8 lib-python/3/distutils/command/install.py
---- a/setuptools/_distutils/command/install.py	Sun Jun 13 10:37:17 2021 +0300
```

Comment:
@mattip do you mind taking  look at the refresh patched?

Reply:
The patch can be made simpler by adding `and os.name != 'windows' at https://github.com/pypa/setuptools/blob/main/setuptools/_distutils/sysconfig.py#L147 and https://github.com/pypa/setuptools/blob/main/setuptools/_distutils/command/install.py#L473

Reply:
Do you think this is "upstreamable?" If not I have to confess I'm not in the mood to re-do the patch :-)

Reply:
It was upstreamed, but upstream adopts it only for pypy>=3.8. We adopted it for pypy>=3.7

Reply:
:confounded: